source/blog/2017-05-12-oo-standalone-registry.md

```diff
@@ -0,0 +1,148 @@
+    [OSEv3:children]
+    masters
+    nodes
+
+    # Set variables common for all OSEv3 hosts
+    [OSEv3:vars]
+    ansible_user=cloud-user
+    ansible_become=true
+
+    openshift_master_default_subdomain=10.8.172.199.xip.io
+
+    openshift_deployment_type=origin
+    openshift_release=v1.4.1
+    deployment_subtype=registry
+    containerized=true
+
+    # enable htpasswd auth
+    openshift_master_identity_providers=[{'name': 'htpasswd_auth', 'login': 'true', 'challenge': 'true', 'kind': 'HTPasswdPasswordIdentityProvider', 'filename': '/etc/origin/master/htpasswd'}]
+    openshift_master_htpasswd_users={'admin': '$apr1$zgSjCrLt$1KSuj66CggeWSv.D.BXOA1'}
+
+    # host group for masters
+    [masters]
+    10.8.172.199
+
+    # host group for worker nodes, we list master node here so that
+    # openshift-sdn gets installed. We mark the master node as not
+    # schedulable.
+    [nodes]
+    10.8.172.199 openshift_node_labels="{'region': 'infra', 'zone': 'default'}" openshift_schedulable=true
+
+This inventory is very similar to the one used in Dusty's blog post about installing OpenShift Origin, so I'm just going to highlight some of the changes I made.
+
+  - Dropped the specification of the `etcd` nodes
+  - Added the `deployment_subtype=registry` entry
+  - Dropped the creation of the `user` username
+  - Skipped the usage of the `openshift_public_hostname` and `openshift_hostname` variables
+  - New values for `openshift_node_labels` and `openshift_schedulable` variables
+
+The most important changes (which are also mentioned in the OpenShift Origin documentation) are the addtion of the `deployment_subtype` and `openshift_schedulable` variables.
+
+The `deployment_subtype` variable instructs the installer to just install the pieces of OpenShift Origin necessary for the stand-alone registry.  Additionally, the `openshift_schedulable` variable allows the single node to be used to run pods.
+
+Run the Installer
+==================
+We are ready to feed the inventory file to `ansible-playbook` and run the installer.  We have to use the same workaround mentioned in Dusty's blog post to enable Ansible to use Python3 during execution.
+
+    $ ansible-playbook -i myinventory -e 'ansible_python_interpreter=/usr/bin/python3'
+    ...
+    ...
+    PLAY RECAP *********************************************************************
+    10.8.172.199               : ok=567  changed=130  unreachable=0    failed=0
+    localhost                  : ok=9    changed=0    unreachable=0    failed=0
+
+Running the installer will take a few minutes, but once it finished, we can get right to the registry console.
+
+Assuming you have used a similar value for `openshift_master_default_subdomain`, you'll be able to access your registry console at a URL similar to `https://registry-console-default.10.8.172.199.xip.io/`.  Just append `registry-console-default.` to your `openshift_master_default_subdomain`.
```

Comment:
Note: the registry console host will be made configurable once I clean up and land https://github.com/openshift/openshift-ansible/pull/4181